docker/daemon.go

```diff
@@ -266,6 +266,13 @@ func (cli *DaemonCli) CmdDaemon(args ...string) error {
 
 	logrus.Info("Daemon has completed initialization")
 
+	// FIXME: perhaps move the following log statement to the end
+	// of daemon.NewDaemon. Why? The exports ExecutionDriver() and
+	// GraphDriver() are only used here outside of the daemon
+	// package. It seems silly to me to export these two just to
```

Comment:
The last part of this is written in the context of the golint PR. When someone comes to FIXME, the context isn't obvious. I would be direct and say what needs changing.